File: app/src/main/java/com/tondz/nguoimu/views/cam_diec/ChonChucNangMuActivity.java
package com.tondz.nguoimu.views.cam_diec;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.tondz.nguoimu.R;

public class ChonChucNangMuActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_chon_chuc_nang_mu);

        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });

        findViewById(R.id.btnCamDiec).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getApplicationContext(), CamDiecActivity.class));
            }
        });
        findViewById(R.id.btnCau).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getApplicationContext(), CamDiecNoiCauActivity.class));
            }
        });
        findViewById(R.id.btnBt).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getApplicationContext(), NguoiBinhThuongActivity.class));

            }
        });


    }
}

File: app/src/main/java/com/phuyen/nguoikhuyettat/views/cam_diec/ChonChucNangMuActivity.kt
package com.phuyen.nguoikhuyettat.views.cam_diec

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.phuyen.nguoikhuyettat.R

class ChonChucNangMuActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.enableEdgeToEdge()
        setContentView(R.layout.activity_chon_chuc_nang_mu)

        ViewCompat.setOnApplyWindowInsetsListener(
            findViewById(R.id.main)
        ) { v: View, insets: WindowInsetsCompat ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        findViewById<View>(R.id.btnCamDiec).setOnClickListener {
            startActivity(
                Intent(applicationContext, CamDiecActivity::class.java)
            )
        }
        findViewById<View>(R.id.btnCau).setOnClickListener {
            startActivity(
                Intent(applicationContext, CamDiecNoiCauActivity::class.java)
            )
        }
        findViewById<View>(R.id.btnBt).setOnClickListener {
            startActivity(
                Intent(
                    applicationContext,
                    NguoiBinhThuongActivity::class.java
                )
            )
        }
        findViewById<View>(R.id.btnGhepTu).setOnClickListener { view: View? ->
            startActivity(
                Intent(
                    applicationContext,
                    GhepTuActivity::class.java
                )
            )
        }
    }
}